src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -327,6 +327,14 @@ internal static SafeSslHandle AllocateSslHandle(SafeFreeSslCredentials credentia
                         Crypto.ErrClearError();
                     }
 
+                    // relevant to TLS 1.3 only: if user supplied a client cert or cert callback,
+                    // advertise that server request the cert post-handshake.
```

Comment:
> advertise that server request the cert post-handshake

Just trying to parse this... is this missing the word "to" after "request"?

Reply:
I think it should be "advertise that to the server". This sets extension in `ClientHello` to indicate willingness to do PHA. (off by default)

Reply:
I have changed the wording to "advertise that we are willing to send the certificate post-handshake". :)